SPECS/kernel/kernel.spec

```diff
@@ -326,6 +326,8 @@ manipulation of eBPF programs and maps.
 %prep
 %setup -q -n CBL-Mariner-Linux-Kernel-rolling-lts-mariner-%{version}
 %patch0 -p1
+# upstream fix: https://github.com/torvalds/linux/commit/d08c84e01afa7a7eee6badab25d5420fa847f783
```

Comment:
commenting again that this should be a patch file instead of a sed.

Reply:
Sure, will convert to patch

Reply:
Will fix post merge

Reply:
Converted to patch